shared/actions/teams.js

```diff
@@ -624,14 +599,10 @@ const _getDetails = function*(action: TeamsGen.GetDetailsPayload): Saga.SagaGene
   )
 }
 
-function _getDetailsForAllTeams(action: TeamsGen.GetDetailsForAllTeamsPayload, state: TypedState) {
-  const actions = state.teams.teamnames
-    .toArray()
-    .map(teamname => Saga.put(TeamsGen.createGetDetails({teamname})))
-  return Saga.sequentially(actions)
-}
+const getDetailsForAllTeams = (state, action) =>
```

Comment:
🐑 this action is unused

Reply:
lets leave this alone for now. how these actions all work is pretty messy and i think a ton of these actions shouldn't exist